build.sh

```diff
@@ -60,7 +60,8 @@ fi
 # Build Java bits: Executor
 if [ "$EXECUTOR_DIR" != $DISABLED_VALUE ]; then
     echo "Building executor in $EXECUTOR_DIR"
-    ./gradlew distZip -p $EXECUTOR_DIR
+    # Retrying gradle command to handle transient failures with downloading jars from public repositories.
+    $REPO_ROOT_DIR/tools/retry -- ./gradlew distZip -p $EXECUTOR_DIR
```

Comment:
This will retry all failures. Could we split the gradle command to first download the prerequisites (possibly with `--refresh-dependencies  Refresh the state of dependencies.`?) and then run the build afterwards?

Reply:
https://github.com/mesosphere/dcos-commons/pull/2345/commits/4581445c82ecc62bf2a3909f0d476ce1775e3cab